### DISCUSSION TITLE
移动云盘挂载问题

### DISCUSSION THREAD

**Discussion by tangwenlongNO1** [AlistGo/alist#7665] [Q&A]
新兑换了黄金会员，然后转存了文件，alist显示不出来怎么解决

**Comment by AQiui** [comment]
我跟你一样,新转存的文件不显示,之前的老文件还可以正常显示

**Comment by pongfcnkl** [comment]
换新个人云，如果还是没有就刷新缓存

**Comment by aaaleileilei** [comment]
我跟你一样,新转存的文件不显示,之前的老文件还可以正常显示，感觉alist挂载的新个人云，手机app和网页登陆进去的个人云。

**Comment by vhuaw** [comment]
我也遇到同样的问题，老文件正常显示，也能播放和使用，新文件显示不出来。在移动云盘里上传的文件，alist挂载里显示不出来，在alist上新建的文件、文件夹alist上能看到，在移动云盘里也显示不出来。
